flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskManagerRunner.java

```diff
@@ -367,12 +365,14 @@ public static TaskExecutor startTaskManager(
 			taskManagerServicesConfiguration.getSystemResourceMetricsProbingInterval());
 
 		TaskManagerServices taskManagerServices = TaskManagerServices.fromConfiguration(
+			configuration,
```

Comment:
I deliberately created the `TaskManagerServicesConfiguration` in order to not having to pass in the `Configuration` object to this method.